### PR TITLE
Default/Additional users & databases

### PR DESCRIPTION
**Pull request by syedrakib**
Some variable name changes to clarify their ***default*** purpose.

Some new variables for ***additional*** purpose.

Please view the PR commit-by-commit for convenience of context.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -57,7 +57,7 @@ module "google_mysql_db" {
   create_timeout      = var.db_timeout
   update_timeout      = var.db_timeout
   delete_timeout      = var.db_timeout
-  user_name           = var.user_name
+  user_name           = var.default_user_name
```
Comment (itsmunim) [inline main.tf]: can we call it root user instead? that's the normal nomenclature. this is a root user and the others are additional users.

Comment (itsmunim) [reply main.tf]: like-

`root_user_name`
`root_user_password`
`root_user_host`

File: variables.tf [hunk 2]
```diff
@@ -50,14 +62,20 @@ variable "db_version" {
   default     = "MYSQL_5_7"
 }
 
-variable "db_charset" {
-  description = "The charset for the MySQL database."
+variable "default_db_name" {
+  description = "Name of the default database to be created."
+  type        = string
+  default     = "default"
```
Comment (sundarsudandiraraj) [inline variables.tf:68]: We could name the default db_name as mysql, to make more sense. 
Just like postgres.

File: main.tf (hunk 1 above)
Comment (syedrakib) [reply main.tf]: Agreed. The Google submodule we used however calls it default - so kept it synonymous with that.

Let me verify if this user indeed gets root privileges assigned to it or not. We can call it root then.

File: variables.tf [hunk 3]
```diff
@@ -20,12 +20,24 @@ variable "private_network" {
 # OPTIONAL PARAMETERS
 # ----------------------------------------------------------------------------------------------------------------------
 
-variable "user_name" {
+variable "default_user_name" {
   description = "The name of the default database user."
   type        = string
   default     = "default"
```
Comment (sundarsudandiraraj) [inline variables.tf]: This could be defaulted to **root** as well.

File: variables.tf (hunk 2 above)
Comment (syedrakib) [reply variables.tf:68]: I just checked. The mysql instances already create a `mysql` db for its internal stuff. So calling this db `mysql` would be conflicting. Let's keep it `default`. This is just the initial db that is created by GCP for users to start using.

File: variables.tf (hunk 3 above)
Comment (syedrakib) [reply variables.tf]: Hi @sundarsudandiraraj I have made this change.

File: main.tf (hunk 1 above)
Comment (syedrakib) [reply main.tf]: Hi @dibosh I have made this change.

**Review by itsmunim — APPROVED** [review]

**Review by itsmunim — APPROVED** [review]

**Pull request merged**